core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/copy.kt

```diff
@@ -1,10 +1,11 @@
 package org.jetbrains.kotlinx.dataframe.api
 
 import org.jetbrains.kotlinx.dataframe.DataFrame
-import org.jetbrains.kotlinx.dataframe.annotations.CandidateForRemoval
+import org.jetbrains.kotlinx.dataframe.util.MESSAGE_SHORTCUT_1_0
+
 // region DataFrame
 
-@CandidateForRemoval
+@Deprecated(MESSAGE_SHORTCUT_1_0, ReplaceWith("columns().toDataFrame().cast()"), DeprecationLevel.ERROR)
```

Comment:
we also usually put the replaceWith contents in the deprecations file. I'm impartial to it, as long as we have at least one constant String there to track the deprecations